lib/awful/widget/watch.lua

```diff
@@ -0,0 +1,62 @@
+---------------------------------------------------------------------------
+--- Watch widget.
+--
+-- @release @AWESOME_VERSION@
+-- @classmod awful.widget.watch
+---------------------------------------------------------------------------
+
+local setmetatable = setmetatable
+local textbox = require("wibox.widget.textbox")
+local timer = require("gears.timer")
+local spawn = require("awful.spawn")
+
+local watch = { mt = {} }
+
+--- Create a textbox that shows the output of a command
+-- and updates it at a given time interval.
+--
+-- @tparam string command The command.
+--
+-- @tparam int timeout (optional) The time interval at which the textbox
+-- will be updated. Default to 5.
```

Comment:
Default parameters work like this, I think: `@tparam[opt=5] int timeout The time interval....`